.ado/templates/e2e-testing-win32.yml

```diff
@@ -1,34 +1,39 @@
 steps:
+  - script: |
+      yarn bundle
+    workingDirectory: apps/win32
+    displayName: 'yarn bundle'
+
+  - script: |
```

Comment:
Is this in additional to WinAppDriver?

Reply:
This installs WinAppDriver (and probably some things which appium uses to talk to WinAppDriver). 2.0 separated the drivers out from the main package to subpackages and you need to explicitly ask appium to install them to run drivers on a particular endpoint. https://github.com/appium/appium/blob/2.0/docs/en/advanced-concepts/migrating-to-appium-2.0.md#bangbang-installing-drivers-during-setup for more info

Reply:
Sounds like macOS will need to install the `xcuitest` driver too. Since we don't have E2E tests in CI, that's not blocking here, but worth calling out. Maybe we need to update our documentation on how to locally run E2E tests now?

Reply:
Ah yeah good call, updated the Windows directions.